hw/js-lesson-3/scripts.js

```diff
@@ -0,0 +1,192 @@
+// 1 Подсчитать сумму всех чисел в заданном пользователем диапазоне.
+
+let num1 = +prompt("enter the first number of range");
+let num2 = +prompt("enter the last number of range");
+let sum = 0;
+ 
+while(num1 <= num2) {       
+       sum += num1;  
+       num1++;
+    }
+ 
+alert(sum)
+
+// 2 Запросить 2 числа и найти только наибольший общий делитель.
+
+let number1 = +prompt("enter 1st number");// 49
+let number2 = +prompt("enter 2nd number");// 7
+let i;
+
+if (number1 > number2) {
+    i = number2; 
+    } else { 
+        i = number1; 
+        }
+while (i < 0) {
```

Comment:
you never pass this condition only if num1 or num2 will be negative, try to log all values in console and look to that